.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
```

Comment:
Well, either use `make` and don't install `latexmk`, or use `latexmk`, I'd say...